dask/array/__init__.py

```diff
@@ -28,3 +28,4 @@
 from ..base import compute
 from .optimization import optimize
 from .creation import arange, linspace, diag
+from .fft import fft, ifft
```

Comment:
If we left this import line off wouldn't `da.fft` just be a module from which we could then call `da.fft.fft`?